shub/image/list.py

```diff
@@ -140,12 +143,17 @@ def raise_shub_image_info_error(error):
     if not isinstance(spiders_list, list):
         raise_shub_image_info_error('spiders section must be a list')
 
+    project_type = metadata.get('project_type')
```

Comment:
@chekunkov @jdemaeyer Please take a qquick look, I've updated code and added some tests.